.travis.yml

```diff
@@ -4,7 +4,6 @@ python:
 sudo: true
 install:
   - "curl -O https://raw.githubusercontent.com/Melissa-AI/Melissa-Core/master/setup/travis-build-core.sh && chmod +x travis-build-core.sh && ./travis-build-core.sh"
-  - "cp data/memory.db.default data/memory.db"
```

Comment:
That sounds good like a solution, @rachmadaniHaryono. Although I am still not sure about what the right way to do this should be. I'll trust your judgement on this one.